app/views/reminder_mailings/new_half_year.html.slim

```diff
@@ -0,0 +1,3 @@
+h1 Errinnerungs Email Versand erstellen
```

Comment:
Typo "E**r**innerung" (also in other places)

Reply:
My bad. thx